chia/cmds/wallet_funcs.py

```diff
@@ -622,3 +622,18 @@ async def execute_with_wallet(
             print(f"Exception from 'wallet' {e}")
     wallet_client.close()
     await wallet_client.await_closed()
+
+
+async def create_did_wallet(args: Dict, wallet_client: WalletRpcClient, fingerprint: int) -> None:
+    name = args["name"]
+    amount = args["amount"]
+    if amount is None:
+        amount = 1
```

Comment:
If we set a default value of 1 in the corresponding `@click.option`, we can drop this check. We might want to replace this with a check that `amount %2 == 1`.

Reply:
Agreed. Regarding the odd number check, we seem to be doing that in `DIDWallet::create_new_did_wallet()` so we'll get an informative exception when that's the case.